src/pages/guides/getting_started/tut.md

```diff
@@ -0,0 +1,1321 @@
+---
+keywords:
+  - Adobe Express
+  - Express Add-on SDK
+  - Express Editor
+  - Adobe Express
+  - Add-on SDK
+  - SDK
+  - JavaScript
+  - Extend
+  - Extensibility
+  - API
+  - Document API
+title: Document API tutorial
+description: This is a Document API in-depth tutorial
+contributors:
+  - https://github.com/undavide
+---
+
+# Document API tutorial
+
+This tutorial will guide you through the creation of your first Express add-on based on the Document API.
+
+## Introduction
+
+Hello, and welcome to this Document API tutorial, where we'll build together a **fully functional Grids System add-on** from scratch. Grid systems are widely used in the design world to bring structure and consistency to all visual content, from flyers to web pages or social media posts.
+
+![](img/tut/grid-addon.png)
+
+Your add-on will allow users to create a variable number of rows and columns, controlling the gutter and color overlays.
+
+### Prerequisites
+
+- Familiarity with HTML, CSS, JavaScript and the Adobe Express add-ons environment. If you need a refresher, follow the [quickstart](/guides/getting_started/quickstart.md) guide.
+- An Adobe Express account; use your existing Adobe ID or create one for free.
+- Node.js version 16 or newer.
+
+### Topics Covered
+
+<!-- List block here -->
+<ListBlock slots="text1, text2" repeat="4" iconColor="#2ac3a2" icon="disc" variant="fullWidth" />
+
+iFrame and Script Runtime communication
+
+Spectrum Web Components
+
+Document API
+
+Color Pickers
+
+Nodes insertion point
+
+Creating and manipulating shapes
+
+Grouping elements
+
+Context permanence
+
+### Getting Started with the Document API
+
+As part of the [Script Runtime](/references/scriptruntime/index.md), the Document API is a powerful tool that extends the capabilities of Adobe Express add-ons, offering direct interaction with the open document. Let's take a moment to review the difference between the two core components of add-ons' architecture.
+
+- The **iFrame** hosts the add-on User Interface and runs its internal logic. You can think about it as a web application operating in a sandboxed environment: it needs to be separate from the rest of the Adobe Express content for security reasons, which is precisely why the add-on is hosted within an `<iframe>` element (a detailed technical description is found [here](/guides/develop/context.md#iframe-sandbox)). If you come from a CEP/UXP background, it's akin to developing the panel of an extension or plugin.
```

Comment:
Good point, Holly.
I tend to use lowercase when referring to the iframe as a tag (e.g., "... inside the `iframe`..." which is a shorthand for  "... inside the `<iframe>`..."), and the camelCase otherwise. Also, because it pairs more nicely with the "Authoring Sandbox", like: "... the Authoring Sandbox and the iFrame", vs. "... the Authoring Sandbox and the iframe".
In the latter, it is as though we're comparing an "environment" with an HTML tag, whereas we're actually comparing two environments—it's a way to elevate the humble iframe to a higher iFrame status 😊
But I reckon it was a totally personal decision, maybe @kerrishotts can weigh in? After all, it's an HZ construct.